Podcast/Player.swift

```diff
@@ -157,7 +161,7 @@ class Player: NSObject {
     func skip(seconds: Double) {
         guard let currentItem = player.currentItem else { return }
         let newTime = CMTimeAdd(currentItem.currentTime(), CMTime(seconds: seconds, preferredTimescale: CMTimeScale(1.0)))
```

Comment:
We are now getting these issues 
<img width="591" alt="screen shot 2017-11-06 at 1 33 47 am" src="https://user-images.githubusercontent.com/14966713/32428205-9db4a952-c292-11e7-8aa7-b87d671d0271.png">
Seems like its due to truncation 
https://github.com/deanm/plask/issues/52

Reply:
Just a warning, will look into after merge.